site/_data/master-toc.yml

```diff
@@ -4,27 +4,33 @@ toc:
       - page: About Velero
         url: /index.html
       - page: How Velero works
-        url: /about
+        url: /how-velero-works
       - page: About locations
         url: /locations
-      - page: Supported platforms
-        url: /support-matrix
   - title: Install
     subfolderitems:
       - page: Overview
         url: /install-overview
       - page: Upgrade to 1.1
         url: /upgrade-to-1.1
-      - page: Quick start with in-cluster MinIO
-        url: /get-started
-      - page: Run on AWS
-        url: /aws-config
-      - page: Run on Azure
-        url: /azure-config
-      - page: Run on GCP
-        url: /gcp-config
-      - page: Restic setup
+      - page: Requirements
+        url: /install-requirements
+      - page: Installing
+        url: /installing
+      - page: Supported providers
```

Comment:
Do you think "supported platforms" is more clear here? I could go with either.

Reply:
What is the difference between platform and provider?

Reply:
I'd consider them roughly synonyms, but I know you disliked "provider" so was looking for alternatives

Reply:
I think minimizing the numbers of terms used for the same thing makes the documentation easier to follow.

As for `provider`, I'm starting to think we need to make more obvious the provider x plugins relationship (a provider has many plugins, a plugin has many providers). WIP.

Reply:
:+1: OK I'm happy with the existing title.

Reply:
I'm still not quite sure where the provider confusion is coming from ...I use it as short hand for "cloud provider."

A cloud provider can have many services. A couple of those services are used by Velero - object storage and volume snapshotting. Thus, a provider could have 1 plugin for each of those 2 services, for a total of 2. These two plugins can both live in the same binary, and the correct one will be invoked by our plugin machinery. Many others could also be in that binary (as seen in Velero's included plugins).

Things such as Portworx and Minio offer only one of those two services - volumes for Portworx, object storage for Minio. Thus they'd each only have 1 plugin, which may or may not be the only plugin within the binary. 

In Velero's API, we don't provide a field for the plugin's type, using the struct type to differentiate. This does mean that solely the names aren't enough to know what a plugin does.